llama_stack/distribution/build_container.sh

```diff
@@ -122,7 +122,7 @@ add_to_docker <<EOF
 # This would be good in production but for debugging flexibility lets not add it right now
 # We need a more solid production ready entrypoint.sh anyway
 #
-ENTRYPOINT ["python", "-m", "llama_stack.distribution.server.server"]
+ENTRYPOINT ["python", "-m", "llama_stack.distribution.server.server", "--template", "$build_name"]
```

Comment:
So docker images built should always be from a registered template?

Reply:
i think this is just the entry point. we can always pass along the args to the actual config, which takes precendece.

Reply:
@yanxi0830 that's why there's a careful `if` in the server code. yaml-config takes precedence even if you always specify template. so if you do `llama stack build my-local-stack` to build your docker and ran it, it won't run unless you give it a yaml (which you naturally will)